scripts/tt_flash_runner.py

```diff
@@ -4,6 +4,14 @@
 
 from runners.core import RunnerCaps, ZephyrBinaryRunner
 from pathlib import Path
+import sys
+from pcie_utils import rescan_pcie
+
+try:
+    import pyluwen
+except ImportError:
+    print("Error, required modules missing. Please run 'pip install pyluwen'")
+    sys.exit(1)
```

Comment:
Won't this throw the same error even without the try except block?

Reply:
It will, yeah- west runners are expected to gracefully handle these errors and print an error message, see here: https://github.com/zephyrproject-rtos/zephyr/blob/main/scripts/west_commands/runners/__init__.py#L15